src/languages/rust.js

```diff
@@ -176,7 +183,7 @@ export default function(hljs) {
     name: 'Rust',
     aliases: [ 'rs' ],
     keywords: {
-      $pattern: hljs.IDENT_RE + '!?',
+      $pattern: IDENT_RE + '!?',
```

Comment:
I don't think any of our literal list of keywords start with `#r` (since I don't see that you've added any keywords) so if that's the case this should be reverted to reduce the work the keyword engine needs to do.

"overmatching" with $pattern for things that will never be found in the keyword lists only slows the engine down.

Reply:
Thanks for the comment!

You're right, there are no keywords starting with `r#`, I think I modified that object without clearly understanding it.
I'll reflect and re-commit!
Thanks.